ArcadiaParties.API/Controllers/UsersController.cs

```diff
@@ -50,10 +53,19 @@ public async Task<IActionResult> GetUsers(CancellationToken cancellationToken)
         [Route("GetCurrentUser")]
         public async Task<IActionResult> GetCurrentUser(CancellationToken cancellationToken)
         {
-            var query = new GetCurrentUserQuery(User);
-            var user = await _mediator.Send(query, cancellationToken);
+            var userQery = new GetCurrentUserQuery(User);
```

Comment:
Misprint.

Reply:
Fixed

Reply:
![image](https://user-images.githubusercontent.com/5681534/77530861-04f34d80-6ea3-11ea-88eb-31169df27529.png)
Where is it?